script/install.sh

```diff
@@ -619,13 +627,13 @@ uninstall_compiler() {
         fatal "$1 is not installed in $path"
     fi
     log "Removing $path/$1"
-    rm -rf "$path/$1"
+    rm -rf "${path:-.}/$1"
```

Comment:
This one is particularly scary - it doesn't look like it can expand to `rm -rf /`, but it gets too close for comfort, e.g. with `--path /`.